File: BackEnd/src/main/java/com/unosquare/admin_core/back_end/viewModels/holidays/HolidayViewModel.java
package com.unosquare.admin_core.back_end.viewModels.holidays;

import com.unosquare.admin_core.back_end.viewModels.employee.EmployeeViewModel;
import com.unosquare.admin_core.back_end.entity.EventStatus;
import com.unosquare.admin_core.back_end.entity.EventType;
import lombok.Data;

import java.time.LocalDate;
import java.util.UUID;

@Data
public class HolidayViewModel {

        private int eventId;

        private LocalDate startDate;

        private LocalDate endDate;

        private EmployeeViewModel employee;

        private EventStatus eventStatus;

        private EventType eventType;

        private boolean isHalfDay;

        private LocalDate lastModified;

        private LocalDate dateCreated;

        private UUID groupId;
}

File: v2/AdminCore.WebApp/Models/Holiday/HolidayViewModel.cs
using AdminCore.WebApi.Models.Employee;
using AdminCore.WebApi.Models.Event;
using AdminCore.WebApi.Models.EventMessage;
using System;
using System.Collections.Generic;

namespace AdminCore.WebApi.Models.Holiday
{
  public class HolidayViewModel
  {
    public int EventId { get; set; }

    public ICollection<EventDateViewModel> EventDates { get; set; }

    public EmployeeViewModel Employee { get; set; }

    public EventStatusViewModel EventStatus { get; set; }

    public EventTypeViewModel EventType { get; set; }

    public bool IsHalfDay { get; set; }

    public ICollection<EventMessageViewModel> EventMessages { get; set; }

    public string LatestMessage { get; set; }

    public DateTime LastModified { get; set; }

    public DateTime DateCreated { get; set; }
  }
}